### PR TITLE
Feature/fsar 35 region variation block

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [x] your work is in a branch based on and synced with the develop branch.
- [x] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [x] your newComponent is imported in the index.js file.
- [x] your folder hierarchy follows the proper structure.
- [x] your code follows the coding standards and naming conventions defined.
- [x] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [x] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [x] your code has been tested in all the different screensizes defined.

### CODE REVIEW

File: src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {% set block_region_string = block_region_first|join(", ") ~ " and " ~ block_region_last %}
+  {% endif %}
+{% endif %}
+
+<div class="regional-variation-block{{ block_region_class is defined ? " " ~ block_region_class : "" }}{{ transparent_background ? "" : " regional-variation-block--solid" }}">
+  <h3 class="regional-variation-block__region">
+    {{ block_region_string }}
+  </h3>
+  <div class="regional-variation-block__copy">
+    {{ wysiwyg_content|striptags('<b>','<a>','<ul>','<ol>','<li>')|raw }}
```
Comment (c-ancia) [inline src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig:19]: Any reason why we remove tags? 🤔

File: src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  </div>
+  {% if files.length >= 1 %}
```
Comment (c-ancia) [inline src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig:21]: I am not sure the files are necessary all in "one block", maybe we'd alternate text & file for instance...

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig (hunk 1 above)
Comment (EmilyHazlehurst) [reply src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig:19]: I didn't want to run |raw on any content so I'm allowing the specified tags only. Happy to just have it be {{ wysiwyg_content|raw}} if it's unlikely to be a risk.

File: src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig (hunk 2 above)
Comment (EmilyHazlehurst) [reply src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig:21]: Do you know how the content will be coming from Drupal? None of the designs have separated files/text so I figured there would be a wysiwyg field and a files field.

Comment (EmilyHazlehurst) [reply src/components/article/RegionalVariationBlock/regionalVariationBlock.html.twig:21]: Will the files be embedded in the wysiwyg? That makes this a more complex ticket if that's the case.

File: src/components/article/RegionalVariationBlock/regionalVariationBlock.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  }
+  &__copy {
+    @include body;
+    color: var(--fsa-primary_black);
+    margin-top: 0.5rem;
+    .document-download {
+      margin-top: 1rem;
+    }
+    .document-download__region {
+      display: none;
```
Comment (c-ancia) [inline src/components/article/RegionalVariationBlock/regionalVariationBlock.scss:58]: aren't we missing the :before also? we don't want the color bar also on any nested files

**Pull request merged**